### ISSUE TITLE
Add Babel support to site build

### ISSUE THREAD

**Issue by Daniel15** [yarnpkg/website#330]
As mentioned in https://github.com/yarnpkg/website/pull/322 - add Babel to this site so we can use more ES6 features.
`make build` should build the JS files.
Need to ensure Netlify is using `make build` to run the build.

**Comment by thejameskyle** [comment]
Do we really have so much that this is necessary?

**Comment by vvo** [comment]
> Do we really have so much that this is necessary?

We do have a bigger amount of code since the merge of #322 (my code), see https://github.com/yarnpkg/website/blob/dbf683453d9128c449cb37db4143e9df2021965f/js/search.js for example.

My POV as a recent contributor. When we did #322 I discovered what the website dev experience was pretty good: automatically rebuilds assets, layouts, that's good!

Then I just wrote some JavaScript the way I write it everyday: with ES2015 semantics but it was not working everywhere because the code was not transpiled. I tried to show the POC search to an unlucky coworker that was using an old chromium and it failed.

So we had to write the code in ES5 and I was a bit surprised that on a Facebook related repository I could not use ES2015 with babel.

Making this website dev experience compatible with yarn itself and being able to `import instantsearch from 'instantsearch.js'` would have helped me here yes. I would have found this flow more inline with my daily flow and the tools I am used to deal with.

The "old chromium" bug is not really the interesting point, I'd just rather prefer to have a more complete frontend tooling to code on yarnpkg.com.

But maybe that's not needed, your call.
